README.md

```diff
@@ -49,15 +49,42 @@ Data is subject to be reaggregated regularly in order to honor Data Subject Acce
 
 ## Data Access and Formats
 
-Data are provided in both Shapefile format as well as Apache Parquet, and can be accessed on Amazon Web Services (AWS) object storage service, S3, at the following paths:
+Data are provided in both Shapefile format as well as Apache Parquet.
 
-* `s3://ookla-open-data/parquet/performance/`
-* `s3://ookla-open-data/shapefiles/performance/`
+### Access via S3
```

Comment:
### Access via S3

The fixed and mobile performance datasets can be found at the following paths on the  `s3://ookla-open-data` bucket, organized by

1. file format (`shapefiles` or `parquet`)
2. service type (`fixed` or `mobile`)
3. year (`2020`)
4. quarter (`1` corresponds to the period starting `2020-01-01`)

```bash
export FORMAT='shapefiles' # (shapefiles|parquet)
export TYPE='fixed'        # (fixed|mobile)
export YYYY='2020'         # 2020
export Q='1'               # 1,2,3 (to date)

aws s3 cp s3://ookla-open-data/${FORMAT}/performance/type=${TYPE}/year=${YYYY}/quarter=${Q}/ . \
--recursive \
--no-sign-request
```

In the example above, the `--recursive` option downloads the compressed shapefiles corresponding to the year, quarter, and service type of the S3 endpoints we passed to the S3 copy command. In this case, `2020-01-01_performance_fixed_tiles.zip`.

To download the mobile and fixed datasets in parquet format for 2020Q3:

``` bash
# Mobile 2020 Q3
aws s3 cp s3://ookla-open-data/parquet/performance/type=mobile/year=2020/quarter=3/2020-07-01_performance_mobile_tiles.parquet

# Fixed 2020 Q3
aws s3 cp s3://ookla-open-data/parquet/performance/type=fixed/year=2020/quarter=3/2020-07-01_performance_fixed_tiles.parquet
```

Reply:
nice! I'll incorporate the explicit keyname definitions in this pr.

Reply:
And I really like the idea of a bash example for using the aws cli. Can you take the lead on a separate pr for that?